src/client/datascience/editor-integration/codewatcher.ts

```diff
@@ -137,6 +138,7 @@ export class CodeWatcher implements ICodeWatcher {
     }
     @captureTelemetry(Telemetry.RunAllCells)
     public async runAllCells() {
+        const iw = await this.getActiveInteractiveWindow();
```

Comment:
Moved the IW fetch to outside of addCode so that multiple calls to addcode wouldn't block on it.